File: HealthSysHub.Web.Pharmacy.UI/src/app/models/changepassword.ts
export interface ChangePassword {
    id?: string;                   // Guid? in C#
    hospitalId?: string;           // Guid? in C#
    staffId?: string;              // Guid? in C#
    labId?: string;                // Guid? in C# (new property)
    pharmacyId?: string;           // Guid? in C# (new property)
    password?: string;
    createdBy?: string | null;
    createdOn?: Date | string | null;
    modifiedBy?: string | null;
    modifiedOn?: Date | string | null;
}

File: HealthSysHub.Web.Utility/Models/ChangePassword.cs
namespace HealthSysHub.Web.Utility.Models
{
    public class ChangePassword
    {
        public Guid? Id { get; set; }                   // Guid? in C#
        public Guid? HospitalId { get; set; }           // Guid? in C#
        public Guid? StaffId { get; set; }              // Guid? in C#
        public Guid? LabId { get; set; }                // Guid? in C# (new property)
        public Guid? PharmacyId { get; set; }           // Guid? in C# (new property)
        public string Password { get; set; }
        public Guid? CreatedBy { get; set; }           // string | null
        public DateTimeOffset? CreatedOn { get; set; }        // Date | string | null
        public Guid? ModifiedBy { get; set; }          // string | null
        public DateTimeOffset? ModifiedOn { get; set; }       // Date | string | null
    }

}
